flink-cyber/flink-indexing/flink-indexing-hive/src/main/java/com/cloudera/cyber/indexing/hive/tableapi/impl/TableApiKafkaJob.java

```diff
@@ -77,6 +83,41 @@ protected void executeInsert(StreamTableEnvironment tableEnv, Map<String, Mappin
     });
   }
 
+  @Override
```

Comment:
Do we need the same checks for Hive tables as well?

Reply:
Moved checks to base class.  Added default conversions from strings to numberics and booleans.